src/Common/src/Interop/Windows/Gdi32/Interop.CombineRgn.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Gdi32
+    {
+        public static class RegionCombineMode
+        {
+            public const int AND = 1;
+        }
+
+        [DllImport(Libraries.Gdi32, ExactSpelling = true, EntryPoint = "CombineRgn")]
+        private static extern int CombineRgnInternal(HandleRef hRgnDest, HandleRef hRgnSrc1, HandleRef hRgnSrc2, int iMode);
```

Comment:
Can you match this up with what I've done in https://github.com/dotnet/winforms/pull/1459?

Reply:
In terms of the enum - we only use RGN_AND. Should I include the other (unused) constants? I opted to do that as I'm guessing the long term view is to mirror this in winforms